projects/vdk-plugins/vdk-jobs-troubleshooting/.plugin-ci.yml

```diff
@@ -30,13 +30,13 @@ build-py311-vdk-jobs-troubleshooting:
   extends: .build-vdk-jobs-troubleshooting
   image: "python:3.11"
 
-#build-vdk-jobs-troubleshooting-on-vdk-core-release:
-#  variables:
-#    PLUGIN_NAME: vdk-jobs-troubleshooting
-#  extends: .build-plugin-on-vdk-core-release
-#  image: "python:3.9"
-
-#release-vdk-jobs-troubleshooting:
-#  variables:
-#    PLUGIN_NAME: vdk-jobs-troubleshooting
-#  extends: .release-plugin
+build-vdk-jobs-troubleshooting-on-vdk-core-release:
```

Comment:
nit: Find the commit message (usually matching PR description) format in `support/git-commit-template.txt`.

Reply:
Hey @ivakoleva ,

I believe this is quite a trivial change, but you are right... Here's a more detailed commit message for completeness:
vdk-jobs-troubleshooting: release the plugin

Why?
Release the first version of the plugin. 

What?
Change CI settings to release the plugin.

Reply:
About testing, did you test it locally, or ci/cd only?